demos/select.scss

```diff
@@ -51,4 +54,29 @@
 .button-container {
   margin: 8px 0;
 }
+
+.demo-example {
+  display: flex;
+  flex: 1 1 auto;
+  flex-direction: column;
+}
+
+.demo-examples-row {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+}
+
+.example {
+  min-width: 300px;
+}
+
+.demo-enhanced-select {
+  width: 400px;
```

Comment:
Yes, it has to be set explicitly or the width will change based on the option selected. It also needs to be added to the menu itself, since the menu is hoisted when the select is initialized. 

I'll add this to the docs.